test/test_cuda/test_mix_bits.py

```diff
@@ -181,5 +172,7 @@ def test_mixed_llmcompressor_format_vllm(self):
         shutil.rmtree(quantized_model_path, ignore_errors=True)
 
 
+
 if __name__ == "__main__":
     unittest.main()
```

Comment:
For AutoRound format, please make sure inference is ready first then supporte it in exporting side